.husky/pre-commit

```diff
@@ -2,3 +2,6 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn lint-staged
+npm test
+npm test
+initial test
```

Comment:
Can we remove these tests from the pre-commit hook? It slows commits down dramatically and throws an error on my machine due to initial not being found? Testing is run once a PR is opened anyway